api/apistruct/struct.go

```diff
@@ -111,6 +111,8 @@ type FullNodeStruct struct {
 		ClientListDeals   func(ctx context.Context) ([]api.DealInfo, error)                                                    `perm:"write"`
 		ClientRetrieve    func(ctx context.Context, order api.RetrievalOrder, ref api.FileRef) error                           `perm:"admin"`
 		ClientQueryAsk    func(ctx context.Context, p peer.ID, miner address.Address) (*storagemarket.SignedStorageAsk, error) `perm:"read"`
+		ClientCalcCommP   func(ctx context.Context, path string, miner address.Address) (*api.CommPRet, error)                 `perm:"read"`
+		ClientGenCar      func(ctx context.Context, ref api.FileRef, path string) error                                        `perm:"write"`
```

Comment:
is this an out path? maybe name it 'out' or 'outpath' if so

Reply:
donesies :)